Mastodon/Protocol/Provider/DataSourceProvider+NotificationTableViewCellDelegate.swift

```diff
@@ -320,9 +304,11 @@ extension NotificationTableViewCellDelegate where Self: DataSourceProvider & Aut
                 return
             }
 
+            guard let account = notification.status?.entity.account else { return }
```

Comment:
❓  No directly related to your changes, but I'm wondering if maybe in such cases we should raise an `assertionFailure()` so we're spotting this in Debug builds? (I have also not yet done this, but maybe we should in the future?) WDYT?